build/linux/Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM portainer/base
 
+LABEL org.opencontainers.image.title="Portainer" \
+    org.opencontainers.image.description="Rich container management experience using Portainer." \
+    org.opencontainers.image.vendor="Portainer.io" \
+    com.docker.desktop.extension.api.version=">= 0.2.0" \
+    com.docker.desktop.extension.icon=https://portainer-io-assets.sfo2.cdn.digitaloceanspaces.com/logos/portainer.png
+
 COPY dist /
+COPY build/docker-extension /
```

Comment:
I guess gruntfile should be changed to run build for extension?

Reply:
the extension ships as part of the image and doesn't require build